<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>executivewindow</class>
 <widget class="QMainWindow" name="executivewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>蛋壳行记</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>500</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 14pt &quot;等线&quot;;</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::East</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>工作手册</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>541</width>
        <height>361</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/image/images/exe2.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bLogout">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>160</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>退出登录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bExit">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>260</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>退出系统</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>380</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Page 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>待处理</string>
     </attribute>
     <widget class="QPushButton" name="bOK">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>400</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>同意</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bNo">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>400</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>不同意</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="decideTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>740</width>
        <height>370</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bRefresh">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>400</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>390</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Page 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>已处理</string>
     </attribute>
     <widget class="QTableWidget" name="doneTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>740</width>
        <height>370</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bRefresh1">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>400</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>390</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Page 3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
